pkg/pack.cmd

```diff
@@ -20,7 +20,7 @@ if exist "%__ProjectDir%\bin" (rmdir /s /q "%__ProjectDir%\bin")
 if not ERRORLEVEL 0 goto :Error
 
 :: Package the assets using Tools
-"%__DotNet%" "%__MSBuild%" "%__ProjectDir%\projects\packages.builds" /p:TargetsWindows=true /verbosity:minimal
+"%__DotNet%" "%__MSBuild%" "%__ProjectDir%\projects\packages.builds" /p:OSGroup=Windows_NT /verbosity:minimal /flp:v=diag
```

Comment:
Can you either remove /flp:v=diag or also add it to the list of arguments in pack.sh?